accepted/0003-npm-link-changes.md

```diff
@@ -0,0 +1,191 @@
+
+#### `npm link remove module-name`
+#### `npm link remove /path/to/module-name`
+
+(Aliases: `npm link rm`, `npm unlink`)
+
+Remove an existing development link from your project.  This does not remove `module-name`—it removes the `link` field from the dependency record and replaces any symlinks to `module-name` with a normally installed copy of it.
+
+#### `npm install`
+
+`npm install --link` is no longer be a valid option. 
+
+`npm install` leaves any development links created by `npm link create` in place.
+
+`npm install module-name@latest` where `module-name` is currently a development link, will update your `package.json` and lock-file but will not do anything on disk.  It warns that the link was left in place.
+
+#### `npm outdated`
+#### `npm update`
+
+Outdated and update run against your lock-file and thus use the unlinked versions when determining what needs doing.  Because `npm update` is built on top of `npm install` it gets the "don't clobber links" behavior automatically.
+
+#### `npm ls`
+
+Listing your install tree reports development linked dependencies as fulfilling their requirements.  In the case where the version of the development linked dependency doesn't match the version in your `package.json` they are flagged visually.
+
+#### `npm pack` and `npm publish`
+
+`npm pack` and `npm publish` fail if any bundled dependencies are currently installed as a development link.
+
+#### `npm unlink`
+
+Is an alias for `npm link clear`.  (Previously this was an alias for `npm rm`.)
+
+### CAVEATS
+
+This specification of the new `npm link` command provides no facility for replacing a specific transitive instance of a dependency.  Concretely:
+
+If you depend on `module-a` and `module-b` and both of them depend on the same version of `module-shared`, there is NO facility for linking `module-a`'s `module-shared` without ALSO linking `module-b`'s `module-shared`.  It SHOULD be possible to build a stand alone tool to support this, but it's out of scope for the initial `npm link` implementation.
+
+## Rationale and Alternatives
+
+Alternative options:
+
+1. Not do anything. This means `npm link` and `npm install --link` with a lock-file are not useful as links will immediately be erased.
+2. Leave all links alone. This would restore previous behavior around `npm link` and `npm install --link` but would make it impossible for npm to do the right thing if a `package.json` dependency changed from a `file:` specifier to a registry one as that would be indistinguishable from a link created with `npm link`.
+
+The second option is appealing for existing users, but introduces an inconsistency with how npm models it's use. npm views its job as synchronizing your `node_modules` with your `package.json` and lock-file.
+
+## Implementation
+
+Broken out by command:
+
+### `npm link`
+### `npm unlink
```

Comment:
Missing backtick here (and is this supposed to be `npm link` & `npm unlink` together (like the heading below)? or is there something missing for `npm link`?)